app/controllers/help_controller.rb

```diff
@@ -0,0 +1,23 @@
+class HelpController < ApplicationController
+  def index
+    @end_user_docs_link = SITE_CONFIG['end_user_docs_link']
+    @organisation_docs_link = SITE_CONFIG['organisation_docs_link']
```

Comment:
Shouldn't these just be called directly from the view? Seems strange to set static links in a controller.

Reply:
`SITE_CONFIG` a constant created during initialization of Rails. Feels decoupling it from what is passed to a view here is a right thing to do. ERB doesn't need to know Rails constants.